types/meteor/globals/reactive-dict.d.ts

```diff
@@ -0,0 +1,15 @@
+declare var ReactiveDict: ReactiveDictStatic;
+declare interface ReactiveDictStatic {
+    new (name?: string, initialValue?: EJSONable): ReactiveDict;
+}
+declare interface ReactiveDict {
+    setDefault(key: string, value: EJSONableProperty): void;
+    setDefault(object: EJSONableProperty): void;
+    set(key: string, value: EJSONableProperty): void;
```

Comment:
Here and in `setDefault(key: string, value: EJSONableProperty): void` the `value` should be optional. [Source](https://github.com/meteor/meteor/blob/71f67d9dbac34aba34ceef733b2b51c2bd44b665/packages/reactive-dict/reactive-dict.js#L82).

Reply:
@radekmie Both methods are overloaded. Do you prefer not?

```
    setDefault(key: string, value: EJSONableProperty): EJSONable;
    setDefault(object: EJSONableProperty): EJSONable;
               ^^^^^^^^^^^^^^^^^^^^^^^^^

    set(key: string, value: EJSONableProperty): EJSONable;
    set(object: EJSONableProperty): EJSONable;
               ^^^^^^^^^^^^^^^^^^
```

Reply:
They should be overloaded, but in the first case the `value` is optional (or at least it seems like it in the [source](https://github.com/meteor/meteor/blob/71f67d9dbac34aba34ceef733b2b51c2bd44b665/packages/reactive-dict/reactive-dict.js#L82)):
```ts
    set(key: string, value?: EJSONableProperty): void;
    set(object: EJSONable): void;
```

Those won't collide as `EJSONable` is an object.

Reply:
Let me know what else to polish @radekmie? Thanks!